_posts/2021-04-26-integrating-airflow-and-okta.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Integrating Airflow with Okta"
+author: kuntalb
+tags:
+- okta
+- airflow
+- featured
+team: Core Platform
+---
+
+
+At Scribd we decided to pair Airflow with Okta. Earlier we were using LDAP for
```

Comment:
This introduction is a bit terse and doesn't really explain much about _why_ we decided to integrate Okta (e.g. better authentication/authorization, MFA, etc) or other details that might help persuade the reader to continue reading the post.

Reply:
added some more info here